docs/source/weights_loading.rst

```diff
@@ -137,6 +138,19 @@ You can manually save checkpoints and restore your model from the checkpointed s
     trainer.save_checkpoint("example.ckpt")
     new_model = MyModel.load_from_checkpoint(checkpoint_path="example.ckpt")
 
+Manual saving with DDP
+======================
+
+Lightning also handles accelerators such as DDP, where multiple processes are used. In this scenario, we only save on the main process.
```

Comment:
Can you elaborate? What does this mean? Can you resume training using a single process? only multi process? Does that change anything about what is being saved?